pkg/operator/ceph/object/user.go

```diff
@@ -134,6 +134,10 @@ func CreateUser(c *Context, user ObjectUser) (*ObjectUser, int, error) {
 		args = append(args, "--caps", rgwAdminOpsUserCaps)
 	}
 
+	if force {
+		args = append(args, "--yes-i-really-mean-it")
```

Comment:
Doubt: 
This is the userName that will be created? 
and is this forcing to create the user?

Reply:
The flag is used in the CLI to force the user's creation: `radosgw-admin user create --uid=foo --yes-i-really-mean-it`.